### DISCUSSION TITLE
Escape forward slash in Java

### DISCUSSION THREAD

**Discussion by Pancor** [cucumber/common#2192] [Q&A Java]
Hi, I have a case that  I want to use Cucumber Expressions so `@ParamterType` will be resolved and also `/` (forward slash) will be interpreted as just a character. When I try to escape `/` like this: `\/` then Java compiler doesn't like it and says `java: illegal escape character`. Example of what I want to achieve:

```java
@Then("endpoint\/path has {status} status")
```
```gherkin
Then endpoint/path has accepted status
```

Is there a way to achieve it?

**Comment by mpkorstanje** [comment]
The way to escape a `\` in a string in Java is to use `\\`. 

So you would write your cucumber expression string as `"endpoint\\/path has {status} status"`.
